src/components/roles/RoleFilterBar.tsx

```
@@ -0,0 +1,30 @@
+            type="text"
+            placeholder="Search role..."
+            className="pl-10 pr-4 py-2 w-full border rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500"
+          />
+        </div>
+        <div className="flex space-x-2">
+          <select className="border rounded-lg px-3 py-2 focus:outline-none focus:ring-2 focus:ring-blue-500">
+            <option>roles</option>
+            <option>Active</option>
+            <option>Inactive</option>
+          </select>
+          <button className="bg-gray-200 hover:bg-gray-300 px-4 py-2 rounded-lg transition">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)